validate_sptx/examples/experiment/field_of_view.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
i don't understand the difference between this and validate_sptx/examples/field_of_view/field_of_view.json

Reply:
As above.